documentation/source/alternativeRunningModes.rst

```diff
@@ -74,7 +74,7 @@ patient's sex needs to be set explicitly with ``PATIENTSEX="male|female|klinefel
      <cvalue name='FAKE_CONTROL_POST' value=".cnv.anno.tab.gz" type='string'
              description="suffix for chromosome wise 1kb coverage files used for fake control workflow"/>
 
-
+Note that if run in no-control mode with SV input (you have ``svOutputDirectory`` set), then ACEseq expects the does not expect the SV file to be named ``svs_${PID}_filtered_somatic_minEventScore3.tsv``, as for the tumor/control case, but ``svs_${PID}_filtered_minEventScore3.tsv``. If you use an output directory of the `Sophia workflow <https://github.com/DKFZ-ODCF/SophiaWorkflow>`_ in no-control mode, you can simply symlink symlink the ``svs_${PID}_filtered_minEventScore3.tsv`` to the created "somatic".
```

Comment:
Note that if run in no-control mode with SV input (you have ``svOutputDirectory`` set), then ACEseq does not expect the SV file to be named as  ``svs_${PID}_filtered_somatic_minEventScore3.tsv`` for the tumor/control case, but ``svs_${PID}_filtered_minEventScore3.tsv``. If you use an output directory of the `Sophia workflow <https://github.com/DKFZ-ODCF/SophiaWorkflow>`_ in no-control mode, you can simply symlink the ``svs_${PID}_filtered_minEventScore3.tsv`` to create the "somatic".